library/Zend/Cache/Backend/Sqlite.php

```diff
@@ -481,10 +481,12 @@ private function _getConnection()
         if (is_resource($this->_db)) {
             return $this->_db;
         } else {
-            $this->_db = @sqlite_open($this->_options['cache_db_complete_path']);
-            if (!(is_resource($this->_db))) {
+            $db = @sqlite_open($this->_options['cache_db_complete_path']);
```

Comment:
I've done this in a few places, it's mainly to prevent the "error" value from being set on the object property by using an intermediate variable to do the "null/false" check on before setting it on the main propery.

This particular one didn't really help (had to ignore anyhow) because the line below doesn't actually throw an exception directly (the method it calls throws the exception), so phpstan doesn't know that execution is halted here.